proteinworkshop/models/graph_encoders/tfn.py

```diff
@@ -169,9 +151,7 @@ def required_batch_attributes(self) -> Set[str]:
     @jaxtyped
     @beartype
     def forward(self, batch: Union[Batch, ProteinBatch]) -> EncoderOutput:
-        """Implements the forward pass of the TFN encoder.
-
-        Returns the node embedding and graph embedding in a dictionary.
+        """Returns the node embedding and graph embedding in a dictionary.
```

Comment:
I think we are now using standardized language for each method's forward function docstring. If you would be able to update this docstring to match the others, that would be great.

Reply:
Done